man/oci-runtime-tool-generate.1.md

```diff
@@ -60,6 +60,14 @@ read the configuration from `config.json`.
   The *fileMode*, *uid*, *gid* are optional.
     *fileMode* is the file mode of the device file.
     *uid*/*gid* is the user/group id of the device file.
+  This option can be specified multiple times.
+
+**--device-remove**=*TYPE:MAJOR:MINOR:PATH[:OPTIONS...]*
```

Comment:
I added this[commit](https://github.com/opencontainers/runtime-tools/pull/325/commits/87bdf5c01e4abda795f240d045740c02327119c0?diff=unified) to fix the problem.